src/index.html

```diff
@@ -5,10 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="header-img-link">
+        <img src="./images/logo.png"
+             alt="MOYO logotype"
+             class="header-img">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-ul">
+          <a href="#" class="nav-link is-active fisrt-nav-link">
+            <li class="nav-li">Apple</li>
+          </a>
+          <a href="#" class="nav-link">
+            <li class="nav-li">Samsung</li>
+          </a>
```

Comment:
This should be the other way around. `ul` needs to have `li` children (which contain links)
```suggestion
          <li class="nav-li">
            <a href="#" class="nav-link">
              Samsung
            </a>
          </li>
```